### PR TITLE
Market Place Back-End

### PR DESCRIPTION
**Pull request by dengsang**
===MARKET PLACE API Introduction ===
## Description
1. I have created a new package backend, under which i have created a marketplace app(API).
2. I have created a skeleton of  what is needed on market place models which include( Users, Ads,  Category, Pricing and Review tables)
3. I have done some partial work on the products(ads) endpoint just to allow the front-end guys to proceed. - will be improving as we start to use the real data.

## Checklist:

- [x ] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [x ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added necessary inline code documentation
- [ ] I have added tests that prove my fix is effective and that this feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by dabusharsystems** [comment]
Awesome Stuff @dengsang Your three files have problems with the codacy linters. I think its pylint. Check your code against pylint for errors i will try check out your code too and check if i can help.

**Comment by dengsang** [comment]
Thanks Patrick,

I will go through it.
Thanks for the feedback

File: .idea/Farmzone.iml [hunk 1]
```diff
@@ -16,7 +16,7 @@
     <content url="file://$MODULE_DIR$">
```
Comment (jimmyaduvagah) [inline .idea/Farmzone.iml]: This file is not part of the projec

**Comment by dengsang** [comment]
Thanks jimmy,

I have git ignored it, i dont why it still appear right there, but lemi fix

**Comment by dabusharsystems** [comment]
I think we can force merge becaue your errors are mostly styling issues.

**Review by dabusharsystems — DISMISSED** [review]
I think we can force merge becaue your errors are mostly styling issues.

**Review by Stephen-Kamau — DISMISSED** [review]

**Review by Stephen-Kamau — DISMISSED** [review]

**Review by dabusharsystems — DISMISSED** [review]
Auto Merge Review

**Review by jimmyaduvagah — APPROVED** [review]
ok

**Review by jimmyaduvagah — DISMISSED** [review]
it is ok from me

**Review by dabusharsystems — DISMISSED** [review]
Only styling issues. No reason not to approve.

**Review by dabusharsystems — DISMISSED** [review]

**Review by dabusharsystems — DISMISSED** [review]
Still trying to merge

**Review by dabusharsystems — DISMISSED** [review]
It checks out now

**Comment by dengsang** [comment]
Good stuff, @dabusharsystems 
Finally zimemerge

**Review by Stephen-Kamau — COMMENTED** [review]
nice work

**Pull request merged**